kahuna/public/js/search/structured-query/query-suggestions.js

```diff
@@ -62,7 +62,9 @@ const subjects = [
 ];
 
 const isSearch = [
-  'staff'
+  'GNM-owned-photo',
```

Comment:
regarding this being Guardian specific and not generic enough - I think it would be better to model this as

```javascript
const isSearch = [{key: 'GNM Owned Photo', value: 'owned-photo', ...}];
```

That is, keep Guardian specific terminology in the rendering tier; Kahuna will make an api request passing `owned-photo` as the value.

This should also mean the UI becomes more human readable with the removal of kebab cased values.

Reply:
> This should also mean the UI becomes more human readable with the removal of kebab cased values.

On the one hand, this is welcomed, although the correct spelling would be `GNM-owned photo` (to follow our style guide for adjectival hyphenation).

On the other hand, there is some value in URL reflecting the values in UI one-to-one. And we have precedents here:

![image](https://user-images.githubusercontent.com/6032869/60170959-72355d80-9801-11e9-8c57-65245d3421fa.png)

Reply:
> keep Guardian specific terminology in the rendering tier

Not sure I follow, would the `company.name` parameter be in Kahuna config or Kahuna would have the GNM name hard-coded?

Reply:
Kahuna would have `company.name` in config. What I meant by:

> keep Guardian specific terminology in the rendering tier

is that the server will have no knowledge of `GNM`, meaning the client will serialise `${company.name} Owned Photo` into the correct value.